---
title: "Research"
---

## Invited Lectures

1. **April 2018, Beijing, China:** *Virus database and online inquiry system based on natural vectors*, Tsinghua University Doctoral Forum in Mathematics
2. **December 2018, Sanya, China:** *A new method to cluster genomes based on cumulative Fourier power spectrum*, [Conferences on Computational and Mathematical Bioinformatics and Biophysics in Tsinghua Sanya International Mathematics Forum](http://tsimf.cn/meeting/detail?id=196_)
3. **June 2019, Beijing, China:** *A novel approach to clustering genome sequences using inter-nucleotide covariance*, [The 8th International Congress of Chinese Mathematicians](https://www.tsinghua.edu.cn/info/1368/81750.htm)
4. **December 2019, Sanya, China:** *A novel approach to clustering genome sequences using inter-nucleotide covariance*, [Conference on Computational and Mathematical Bioinformatics and Biophysics in Tsinghua Sanya International Mathematics Forum](http://www.tsimf.cn/meeting/detail?id=154)
5. **July 2021, Chongqing, China:** *Statistical methods for Alzheimer's disease gene-mapping in large cohort studies*, The 1st Youth Forum of [Peking University Chongqing Institute of Big Data](https://bda.pku.edu.cn/ydkyjg/bjdxzqdsjyjy.htm)
6. **December 2021, Sanya (online), China:** *Full chromosomal relationships between populations and the origin of humans*, [Conference on Computational and Mathematical Bioinformatics and Biophysics in Tsinghua Sanya International Mathematics Forum](http://tsimf.cn/meeting/detail?id=284)
7. **April 2022, Nanjing (online), China:** *Assessing the transmissibility of the new SARS-CoV-2 variants: from Delta to Omicron*, [Forum for Young Scholars in Mathematics and Mathematical Physics at Shing-Tung Yau Center of Southeast University](https://yauc.seu.edu.cn/2022/1208/c27640a430948/page.psp)
8. **August 2022, Nanjing, China:** *Assessing the transmissibility of the new SARS-CoV-2 variants: from Delta to Omicron*, [The 9th International Congress of Chinese Mathematicians](http://iccm.tsinghua.edu.cn/iccm2022/#/)
9.  **November 2022, Beijing, China:** *Case-only design to estimate interaction effect in GWAS of rare disease*, [Postdoc Workshop for Yau Mathematical Sciences Center, Tsinghua University](https://ymsc.tsinghua.edu.cn/en/info/1060/2473.htm)

## Poster Presentations

1. **December 2015, Suzhou, China:** *Microarray-based Cancer Prediction Model via One-Gene*, [Cold Spring Harbor Asia (CSHA) / American Association for Cancer Research (AACR) joint conference on Big Data, Computation and Systems Biology in Cancer](https://www.csh-asia.org/?content/1525)
2. **May 2023, New York:** *The rare disease assumption in the case-only design*, [Postdoc Research Symposium at Columbia University](https://research.columbia.edu/sites/default/files/content/OPA/2023%20Columbia%20University%20Postdoctoral%20Research%20Symposium%20Abstracts%20(2)-1-30.pdf)
3. **November 2023, Washington DC:** *The rare disease assumption in the case-only design*, [The 75th American Society of Human Genetics](https://www.ashg.org/meetings/2023meeting/)

## Journal Reviewer 

- *Frontiers in Medicine*
- *Frontiers in Neuroscience*
- *Journal of Infection and Public Health*
- *Clinical Epidemiology and Global Health*
- *Communications in Information and Systems*

## Publications

### Original Articles (Peer Reviewed)

1. **Dong R**, Zheng H, Tian K, Yau SC, Mao W, Yu W, Yin C, Yu C, He RL, Yang J, Yau SST (2017) Virus database and online inquiry system based on natural vectors. *Evolutionary Bioinformatics* 13:1-7. [PMID: 29308007](https://pubmed.ncbi.nlm.nih.gov/29308007/); [PMC5751915](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5751915/)
2. **Dong R**\*, Zhu Z\*, Yin C, He RL, Yau SST (2018) A new method to cluster genomes based on cumulative Fourier power spectrum. *Gene* 673:239-250. [PMID: 29935353](https://pubmed.ncbi.nlm.nih.gov/29935353/) \*Co-first author
3. Pei S, **Dong R**, He https://pubmed.ncbi.nlm.nih.gov/RL, Yau SST (2019) Large-scale genome comparison based on cumulative Fourier power and phase spectra: central moment and covariance vector. *Computational and Structural Biotechnology Journal*, 17:982-994. [PMID: 31384399](https://pubmed.ncbi.nlm.nih.gov/31384399/); [PMC6661692](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6661692/)
4. **Dong R**, He L, He RL, Yau SST (2019) A novel approach to clustering genome sequences using inter-nucleotide covariance. *Frontiers in Genetics*, 10:234. [PMID: 31024610](https://pubmed.ncbi.nlm.nih.gov/31024610/); [PMC6465635](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6465635/)
5. Sun N\*, **Dong R**\*, Pei S, Yin C, Yau SST (2020) A new method based on coding sequence density to cluster bacteria. *Journal of Computational Biology*, 27(12): 1688-1698. [PMID: 32392428](https://pubmed.ncbi.nlm.nih.gov/32392428/) \*Co-first author
6. He L\*, **Dong R**\*, He RL, Yau SST (2020) A novel alignment-free method for HIV-1 subtype classification. *Infection, Genetics and Evolution*, 77: 104080. PMID: 31683009 \*Co-first author
7. He L, **Dong R**, He RL, Yau SST (2020) Positional correlation natural vector: a novel method for genome comparison. *International Journal of Molecular Sciences*, 21(11): 3859. [PMID: 32485813](https://pubmed.ncbi.nlm.nih.gov/32485813/); [PMC7312176](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7312176/)
8. Pei S, **Dong R**, Bao Y, He RL, Yau SST (2020) Classification of genomic components and prediction of genes of Begomovirus based on subsequence natural vector and support vector machine. *PeerJ*, 8:e9625. [PMID: 32832270](https://pubmed.ncbi.nlm.nih.gov/32832270/); [PMC7409808](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7409808/)
9. **Dong R**\*, Pei S\*, Yin C, He RL, Yau SST (2020) Analysis of the hosts and transmission paths of SARS-CoV-2 in the COVID-19 outbreak. *Genes*, 11(6): 637. [PMID: 32526937](https://pubmed.ncbi.nlm.nih.gov/32526937/); [PMC7349679](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7349679/)
10. **Dong R**, Pei S, Guan M, Yau SC, Yin C, He RL, Yau SST (2022) Full chromosomal relationships between populations and the origin of humans. *Frontiers in Genetics*, 12: 828805. [PMID: 35186019](https://pubmed.ncbi.nlm.nih.gov/35186019/); [PMC8847220](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8847220/)
11. **Dong R**\*, Hu T\*, Zhang Y, Li Y, Zhou XH (2022) Assessing the transmissibility of the new SARS-CoV-2 variants: from Delta to Omicron. *Vaccines (Special Issue: COVID-19: Epidemiology and Transmission)*, 10(4): 496. [PMID: 35455246](https://pubmed.ncbi.nlm.nih.gov/35455246/); [PMC9026126](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9026126/)


## Under Revision
- **Dong R**, Wang TG, DeWan AT, Leal SM (2024) The case-only design: a powerful approach to detect interactions that should be used with caution: the impact of disease prevalence and other factors on type I and II errors. Submitted.
- Li Y, Tai QC, **Dong R**, You YA, Liang MF, Zhou XH (2024) A 2-transcript signature discriminated SARS-CoV-2 infections from common respiratory viruses before detectable viral RNA on RT-PCR testing (submitted)